git-work.sh

```diff
@@ -75,6 +75,10 @@ case "$1" in
     git fetch origin pull/"$2"/head:pr/"$2"
     git checkout pr/"$2"
     ;;
+  install)
+    path=$(pwd)/git-work.sh
+    git config --global alias.work '!sh "$path" '
```

Comment:
@fabriciofx didn't work.

First, I tried like you wrote `$ git-work.sh install` but I got an error `bash: git-work.sh: command not found`.

Then, I realise that I'm on Windows so I've tried this `$ bash git-work.sh install` but I received the "help" information. In other words, `install` didn't run.

To solve this we need to change this checking [here](https://github.com/mdbs99/git-work/blob/master/git-work.sh#L40). This line is testing 2 arguments but `install` has just one.

Finally, I tried this one `$ bash git-work.sh install foo` and looks like worked, but not. My *alias* was recorded like this:

```ini
[alias]
	work = "!sh \"$path\" "
```